python/vtool/data.py

```diff
@@ -1021,35 +1034,16 @@ def _get_influences(self, folder_path):
         except:
             return
 
-        found_single_file_weights = False
-
-        influences = []
-
-        for filename in files:
-
-            if filename == 'all.skin.weights':
-                found_single_file_weights = True
-                continue
-
-            if not filename.endswith('.weights'):
-                continue
-
-            influences.append(filename)
+        found_single_file_weights = any(filter(lambda x: x == 'all.skin.weights', files))
+        influences = [filename for filename in files if filename.endswith('.weights')]
```

Comment:
Careful, it has a continue for the original iterator, add an and to the comprehension.

Reply:
resolved.